db/migrate/20160602143125_add_reject_mail_at_mta_to_info_request.rb

```diff
@@ -0,0 +1,10 @@
+# -*- encoding : utf-8 -*-
+class AddRejectMailAtMtaToInfoRequest < ActiveRecord::Migration
+ def up
+    add_column :info_requests, :reject_mail_at_mta, :boolean, :default => false, :null => false
```

Comment:
Might be worth calling this `reject_incoming_at_mta` to match the language used elsewhere (e.g. `IncomingMessage`/`OutgoingMessage`, `:rejected_incoming_count`)